docs/migration-guide-2.5-2.6.md

```diff
@@ -2,12 +2,23 @@
 title: Migration Guide 2.5-2.6
 ---
 
-Due to the enablement of the telemetry component by default, some logging resources must be deleted. When you upgrade from Kyma 2.5 to 2.6, either run the script [2.5-2.6-cleanup-logging-fluent-bit.sh](./assets/2.5-2.6-cleanup-logging-fluent-bit.sh) or run the commands from the script manually.
+# Migrate Fluent Bit
 
-If you are using override values for the logging component to configure additional Fluent Bit filters, parsers, or outputs, you can apply them by creating a LogPipeline or LogParser custom resource. For more information, see [Kyma's telemetry component](./01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md). Other overrides to Fluent Bit must be applied to the telemetry component now.
+With this release, the Telemetry component is enabled by default; so Fluent Bit configuration is now handled by the Telemetry component instead of the Logging component.  
```

Comment:
```suggestion
With this release, the Telemetry component is enabled by default; so Fluent Bit is now managed by the Telemetry component instead of the Logging component.  Hereby, the configuration of FluentBit is done now at runtime via the new LogPipeline CRD instead of overrides at installation time.
```